### PR TITLE
chore(cketh/ckerc20): proposal to upgrade ledger suite orchestrator and managed canisters

### PR DESCRIPTION
**Pull request by gregorydemay**
Proposal to upgrade all ledger suites managed by the ledger suite orchestrator to the latest version.

### CODE REVIEW

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_12_02.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Proposal to upgrade the ledger suite orchestrator canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `2190613d3b5bcd9b74c382b22d151580b8ac271a`
+
+New compressed Wasm hash: `57b63457b2721e7fe649fe418576236f7a5ca49669f1acae208880a84011f167`
```
Comment (ninegua) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_12_02.md:7]: Verified!

Comment (THLO) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_12_02.md:7]: Verified!

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_12_02.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+Upgrade all ledger suites managed by the orchestrator to the latest version ([ledger-suite-icrc-2024-11-28](https://github.com/dfinity/ic/releases/tag/ledger-suite-icrc-2024-11-28)) to continue the migration towards stable memory.
+
+
+## Upgrade args
+
+```
+git fetch
+git checkout 2190613d3b5bcd9b74c382b22d151580b8ac271a
+cd rs/ethereum/ledger-suite-orchestrator
+didc encode -d ledger_suite_orchestrator.did -t '(OrchestratorArg)' '(variant { UpgradeArg = record {git_commit_hash = opt "2190613d3b5bcd9b74c382b22d151580b8ac271a"; ledger_compressed_wasm_hash = opt "9637743e1215a4db376a62ee807a0986faf20833be2b332df09b3d5dbdd7339e"; index_compressed_wasm_hash = opt "d615ea66e7ec7e39a3912889ffabfabb9b6f200584b9656789c3578fae1afac7"; archive_compressed_wasm_hash = opt "2d25f7831894100d48aa9043c65e87c293487523f0958c15760027d004fbbda9"}})' | xxd -r -p | sha256sum
```
[the remark this answers is not in the document]
Comment (ninegua) [reply rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_12_02.md:28]: Hashed of all three wasms verified!

**Review by ninegua — APPROVED** [review]

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_12_02.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+```
+Verify that the hash of the gzipped WASM for the ledger, index and archive match the proposed hash.
+
+```
+git fetch
+git checkout 2190613d3b5bcd9b74c382b22d151580b8ac271a
+"./ci/container/build-ic.sh" "--canisters"
+sha256sum ./artifacts/canisters/ic-icrc1-ledger-u256.wasm.gz
+sha256sum ./artifacts/canisters/ic-icrc1-index-ng-u256.wasm.gz
+sha256sum ./artifacts/canisters/ic-icrc1-archive-u256.wasm.gz
```
Comment (THLO) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_12_02.md:106]:
```
$ sha256sum ./artifacts/canisters/ic-icrc1-ledger-u256.wasm.gz
9637743e1215a4db376a62ee807a0986faf20833be2b332df09b3d5dbdd7339e
```
```
$ sha256sum ./artifacts/canisters/ic-icrc1-index-ng-u256.wasm.gz
d615ea66e7ec7e39a3912889ffabfabb9b6f200584b9656789c3578fae1afac7
```
```
$ sha256sum ./artifacts/canisters/ic-icrc1-archive-u256.wasm.gz
2d25f7831894100d48aa9043c65e87c293487523f0958c15760027d004fbbda9
```
:+1:

**Pull request merged**